app/common/uuid.py

```diff
@@ -0,0 +1,73 @@
+import typing
+import uuid
+from enum import Enum
+
+from app.errors.errors import InvalidUsageError, NotInDatabaseError
+from app.models import Sessions, Scores, Users, Conversations, AlignmentScores
+
+
+class uuidType(Enum):
```

Comment:
This has to do with the conversation dashboard. I don’t know if I’d consider it ‘common’ to be honest!